procurementSms/tables/procurements.yml

```diff
@@ -0,0 +1,112 @@
+type: table
+key: procurements
+name: Procurements
+itemName: Procurement
+summary:
+  title: "Procurement {{title}}"
+  subtitle: "Created at  {{createdAt}}"
+
+permissions:
+  requester:
+    create: false
+    view: true
+    edit:
+      own: false
+      assigned: false
+    delete: false
+  manager:
+    create: false
+    view: true
+    edit:
+      own: true
+      assigned: true
+    delete: true
+  purchaser:
+    create: false
+    view: true
+    edit:
+      own: false
+      assigned: false
+    delete: false
+  accountant:
+    create: false
+    view: true
+    edit:
+      own: false
+      assigned: false
+    delete: false
+
+fields:
+  createdAt:
+    type: datetime
+    label: Created At
+  approvedAt:
+    type: datetime
+    label: Approved At
+  approved:
+    type: boolean
+    label: Approved
+  requester:
+    type: user
+    label: Requester
+  manager:
+    type: relationship
+    label: Manager
+    table: managers
+    required: true
+  selectedManager:
+    type: user
+    label: Selected Manager
+    readOnly: true
+  accountant:
+    type: user
+    label: Accountant 
+  purchaser:
+    type: user
+    label: Purchaser
+  approvedQuotation:
```

Comment:
Debería ser referencia a un quote específico.

Reply:
(es decir, no debería ser un texto, sino un reference). Aquí veo necesidad de un feature nuevo: Los filtros que actualmente tenemos para la tabla, poder tenerlos también para los relationships. (Para poder filtrar qué quotes pueden seleccionar).

Reply:
Si me encontre con ese problema. Define los Quotations como un tableField pero no hay como selecionar uno o varios campos de esa tabla. en algún otro field. 
Por eso por ahora le puse como "text" y este es el que aparece en el formulario del outcome "approve" en la tarea "Analyze and Approve Quotations"